mappings/b88ea544b60f2e018b198dfce0a40774172ce2dc4cfdfb1f7ddd0c275065746572.json

```diff
@@ -0,0 +1 @@
7NnR1F4f0msMEk8mdnVq+c0a960ZatWjRs0yspIdxqG4CyRYEnTZIJLggQEghRSAgGN6rqmG4bucDgPZ378jW+Gc/7GG29MmTJFCBGPxy+99NIHHnhAlS6qooazzjpr5MiRjLG/J5B/A4WFhddcc82rr77q9XpP9EBk5YiK1+tNT0+v7LkdCcXFxYrauLwFrx6GisCozxljytesdLbSI4GA4CiZx+N1uDy79uyd/+WCxV8v3LV5g2HFs9KCmW7D49IpImcsyWRSyIhlR2Lx4rhpc6n7U9KzqmVWq9moadPGTRrXrFGzWnrQ5/EgQdtMxpKWbTMhBQBKKhCQSEkACaWUak6nwzAchmEc0l93jFC3v3fv3q1bt2ZnZzdo0EB9omRj0qRJO3bseOWVV46nqAghhg4deueddzZt2vREF5V/tVl/BASDQafTWVxcbJrmQV2QpJRlZMeHTCMgIIAggB6NJyLRWMDnvmLIpSMuvXx/QcHmXdu37NqRu3t/LBoRkhPEVJfT6XJ7vP601LTstEBGZlpKMCUQ8OsalUKYpmlblmmZ+clEKdOulABAEAgBgrrhMByGw3A6dE2n9HdxrQozV3k7qlWrVq1aNQDgnJfVYANA586dv/nmG7XDHJ9Vq0yj1NTU9evXN23a9C97JfzLcaKKCgA4nc6srKxEIhGPx5PJpBCifNdvAFDh6rJP1O/lJYcQAoQwxotCIUmY4TY6tGzWpU0bqR3IS0chEEFKyYVggnNkzOK2HSkKcRDwe5oCUS5sSoimaUqnUj8PtziqaKWWxZ2gXGOCsqSskpKS3bt3Kz9hVYx+ENQo1atX37hxI1TN2+F44gQWFQBARLfb7Xa7hRCmaarQvtK44IBrTjGFa5omhEgkEmWu5IOyZgAp5xiOJUDGUZRyYwmUVNLSLBUCCFwiAhKpEeTKlYyU0jLBcDgcVaFZHe138ucPpZTp6ekZGRmrVq1SonJ8ksEAoE6dOitWrDjcxE4gVFoOWKVc52+DEOJyuVwu15EP83q9Ku84mUzatq1IxA/88QDBCUGgUhn1BJBIKSVIkCCkRAKIBAmlVHNoDsNQYcE/bx3/Kr28zFypVavWqlWr+vfvf3z8UWWi8tFHH8ExF//94zixd5W/AU3T/H6/3+/nnCuSJCUznHPlGAApVaahetLKH0CppumaRjVdL92kjvzg/z1yUh6tWrVasmQJHK/plYmKaZrxeNztdv+r3iBHi/+RXeVvQDnKHA5H+Q/lAZR9clCmzEEHwwly70qwmzVrNnv27ONmXuMBAlGn05mfn1+7du3jMGjV4cTeEysdymNGy6G8l/nPB58QcgIHVm2jRo1CoZBqx3ncLHtd130+344dO47boFWE/0Tl/xFq1KgRCAS2b98Ox2vVKiL27OzsXbt2HbdBqwj/icr/CyhfOaU0Ozt769atcHxXbVZWVvleOicoKkFUTmhb7f8P1As+NTV106ZNcLxERdlI6enpe/bsOQ7DVSn+21X+f6FWrVpKATueyMnJyc3NhRPEBXI4/Ccq/5tQ2ZPKm6f2E7VM69atm5+fD+Vi+VUKNahKqihLtDlBcQJP/T8cDmVpo2WEBGV/qlGjRmFhoSqPO27zSU9P55xHo1E4kS37ymHCP3Hv/38PpeSa8fiMGTNUwLFnz57jxo1Tzyg7OzsSiUQikWAweByMTHV9n88HAHl5eYFA4MS1bP/fRev/P8A0zSFDhuTl5Y0dO1YIMXXq1C1btjz11FMAoMqK8vLyjo+oQLnQSmFhYcOGDat6uKpDpTW4q5Tr/IdjhMp7nz9//rp161asWBEMBgGgV69enTp1Wrd+fbOmTb0+n8vjzsvLa9So0fGZkhLItLS0UCh0fEasIvxnq/wPYufOnTVr1nS73cXFxQDgdrsNw4hEIgCga1owEMzLzz9uk1GVEenp6cczS6AqcKyicoLqnf+rUC6m008/fcuWLU8//XQsFtu5c+fo0aODwWDrVq2UNe/3+wsK8uHAIj4+SEtLO9GjkJWjgB3PL/0/HAHqzdW4ceOpU6dOmDDhtddeSyQSGRkZL7/8stPpNG2LUurz+fILCo7zlFJSUk70KOSxioraT/8TlX8VhBD9+/c/5ZRT1qxZ43Q6O3To4HK5pJQUCQCkpKQVFR4/s0GJSkZmhmVbx23QqsB/Zv3/IFTZcHZ2dnZ2NhyoLChTlYPBwO5du+G4kdMjAkB6RkY8kYATucCrEub9n7nyL4QKdpWx2JR/Rl6PJ5FMqIOOx0wAACAYDBYVFdmMnbhRuMoR8RP05v+3oWpvyguJ+t3hcDBmH+fJ+Hw+0zSTZvKvD/23otJE5T9z5USBYRilXSePozbgNBwIYCb+34tKaVX6fzgRUG6fOX4V9pqmgYRYLHYcRqwi/B9jCGuGP92+lgAAAABJRU5ErkJggg==", "signatures": [{"signature": "f102a6c722b9561e40e01d35526677f6e3cecde93c0783c227b0b18957b38542d9c40266e7217c65b06e591b71bad97d9e4563baf7493a73d4bc24790c886d0f", "publicKey": "1d27f2770e28963ba94b80a35911e4e42e2edf238600c51932f6209efe0344bd"}]}, "description": {"sequenceNumber": 0, "value": "Peter the peoples memecoin ", "signatures": [{"signature": "ce31d433a914acfe1a2546a2cbe31f351e4e867e569202007e12f64d2180141fab9aadfe7536f18e3ce6ede4fd065d576db554f1c135807425cbf81979bf1600", "publicKey": "1d27f2770e28963ba94b80a35911e4e42e2edf238600c51932f6209efe0344bd"}]}}
```

Comment:
Hi @CardanoTokenIO,

can't be approved due to CR issues. Change the logo and update the PR.

Thanks!